doc/source/tutorials/emulator.rst

```diff
@@ -0,0 +1,182 @@
+                            "acquisition": {
+                                "kind": "acquisition",
+                                "duration": 100.0
+                            },
+                        "probe": {
+                            "duration": 100.0,
+                            "amplitude": 0.1,
+                            "envelope": {
+                                "kind": "gaussian_square",
+                                "rel_sigma": 0.2,
+                                "width": 0.75
+                                },
+                            "relative_phase": 0.0,
+                            "kind": "pulse"
+                        }
+                        }
+                    ]
+                    ],
+                    "CP": null
+                }
+                }
+                }
+            }
+
+
+We are defining an Hamiltonian with a transmon with two-levels, with a frequency of :math:`\omega_q / 2 \pi = 5 \ \text{GHz}` and
+anharmoncity :math:`\alpha/2 \pi = - 200 \ \text{MHz}`,
+with :math:`T_1 = 1 \  \mu s` and :math:`T_2 = 1.9 \ \mu s`.
+Everything else follows the usual Qibolab conventions. Keep in mind that you still need to define also a readout pulse even if all
+parameters except for the duration are ignored in the current emulator.
+
+We are now going to give an example on how to setup the `platform.py` file.
+
+
+.. testcode::  python
+
+    # emulator / platform.py
+
+
+    from pathlib import Path
+    from qibolab import ConfigKinds, IqChannel, Platform, Qubit
+    from qibolab.instruments.emulator import EmulatorController, HamiltonianConfig
+
+    FOLDER = Path.cwd()
+    ConfigKinds.extend([HamiltonianConfig])
+
+
+    def create() -> Platform:
+        """Create a dummy platform using the dummy instrument."""
+        qubits = {}
+        channels = {}
+
+        for q in range(1):
+            qubits[q] = qubit = Qubit.default(q)
+            channels |= {
+                qubit.drive: IqChannel(mixer=None, lo=None),
+            }
+
+        # register the instruments
+        instruments = {
+            "dummy": EmulatorController(address="0.0.0.0", channels=channels),
+        }
+
+        return Platform.load(
+            path=FOLDER,
+            instruments=instruments,
+            qubits=qubits,
+        )
+
+We can observe that we need to allocate an object of type ``EmulatorController`` where we load the channels.
+Note that in order to enables the config to support the Hamiltonian configuration we are adding it explicitly
+in the statement ``ConfigKinds.extend([HamiltonianConfig])``.
+
+If the platform and the configuration have been coded correctly it is possible to use the emulation
+platform exactly as any other platform. If we store both the ``parameters.json`` and ``platform.py`` in a directory named
+``emulator`` we can load it using
+
+.. code-block:: python
+
+    from qibolab import create_platform
+
+    platform = create_platform("emulator")
```

Comment:
In principle, this is redundant, since it should be already discussed for a general platform.
Moreover, this platform creation mode is quite optional. Without `Hardware`, you just need to run the function in the script to get a platform. And with `Hardware`, you need to run the function and pass it to the `Platform` constructor, together with a path.

However, I'm not strongly against it. But, in case, the `QIBOLAB_PLATFORM` env var should be mentioned as well, since required.

Reply:
In the comment above I kept just `Hardware` so that I can remove this section.

Reply:
Yes, definitely. Let me focus on merging a basic version of #1088 and #1175, and I'll take some time to overhaul the whole Qibolab documentation. We definitely need it at this point.